src/devicelock/devicelock.cpp

```diff
@@ -199,7 +199,9 @@ bool DeviceLock::runPlugin(const QStringList &args)
     qWarning() << process.readAllStandardError();
 #endif
 
-    return process.exitCode() == 0;
+    if (process.exitCode() == 0) return OK;
```

Comment:
Please don't - that would mean `if (!runPlugin(...)) { /* plugin returned success */ }`, which seems more error-prone.